R/g_mlm.R

```diff
@@ -94,23 +113,29 @@ g_mlm <- function(mod, p_const, mod_denom = mod, r_const = NULL, infotype = "exp
     stop("g_mlm() only available for lme or gls models. Please specify such a model in the 'mod_denom' argument.")
   }
 
-  theta <- extract_varcomp(mod_denom)              # full theta vector
+  theta <- extract_varcomp(mod_denom)
```

Comment:
Is there a reason we need to call `extract_varcomp()` twice? Would it work to just pass `separate_variances = separate_variances`?

Reply:
Yes, we can just pass `separate_variances = separate_variances`. I do not know what I was thinking...